YouTubeSpammerPurge.py

```diff
@@ -182,9 +182,9 @@ def get_replies(parent_id, video_id):
 # Then uses print_prepared_comments() to print / log the comments
 def print_comments(comments, logMode):
   j = 0 # Index when going through comments all comment groups
-  if len(comments) > 50:
-    remainder = len(comments) % 50
-    numDivisions = int((len(comments)-remainder)/50)
+  if len(comments) > 1000:
+    remainder = len(comments) % 1000
+    numDivisions = int((len(comments)-remainder)/1000)
```

Comment:
Why change this?

Reply:
Especially without changing the next few lines also.